config/routes.rb

```diff
@@ -111,8 +111,7 @@ def not_implemented
               end
 
     get '/:page_type/:id/preview' => 'articles_preview#show',
-        page_type: /articles|action_plans|news|videos|corporate|tools/,
-        constraints: ValidResource.new(:article)
```

Comment:
There are move valid resource call. I will remove all blacklisted articles Valid resource that is being called.